recipes/prismatic/cmakelists.patch

```diff
@@ -0,0 +1,17 @@
+diff --git CMakeLists.txt CMakeLists.txt
+index c7791812..767d4199 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -29,7 +29,11 @@ set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
+ 
+ if (UNIX)
+ 	set(CMAKE_LIBRARY_PATH ${CMAKE_LIBRARY_PATH} /usr/local/lib)
+-	link_directories(${CMAKE_LIBRARY_PATH})
++	if(DEFINED ENV{CONDA_PREFIX})
++		link_directories(${CMAKE_LIBRARY_PATH} $ENV{CONDA_PREFIX}/lib )
++	else (DEFINED ENV{CONDA_PREFIX})
++		link_directories(${CMAKE_LIBRARY_PATH} )
++	endif (DEFINED ENV{CONDA_PREFIX})
```

Comment:
I think this patch is not necessary, you can explicit set the ``CMAKE_LIBRARY_PATH`` when invoking ``cmake``

Reply:
In testing your suggestion I've found that I don't need the patch anymore (originally I added it to deal with dependencies for local builds - something along the way has changed so that it is no longer needed). So I've removed the patch file completely.